### ISSUE TITLE
part segmentation

### ISSUE THREAD

**Issue by zaiweizhang** [WangYueFt/dgcnn#8]
Hi,

I am trying to reproduce your results showing in the paper with your code but I am not able to do it.
Would you mind releasing your trained model for shapenet part segmentation task?

Thanks!

**Comment by syb7573330** [comment]
Hi, 

Thanks for asking. The reported result is the best result during the training process, and it may not be the result from the last epoch. Could you also save intermediate models, and try them out?

**Comment by zaiweizhang** [comment]
Hi,

Thanks for replying! I have used two gpus in the training but I am not able to use batch size 16 due to the GPU memory issue (I have two 1080Ti). The batch size I used is 10 per GPU. And, I have tried different models trained at different epochs. I can get around 84.7%. It may be the batch size issue.

Is there any concern about releasing the pre-trained model? 
Sorry for all the inconvenience! It's just not very easy for me to get around the hardware issue.
If you have the pre-trained model stored somewhere, would you mind sharing it to me via google drive or some other resource?

Thanks!

**Comment by syb7573330** [comment]
Hi, 

I don't remember if I have any pre-trained model stored on our lab computer. Let me re-run the model when I get time.

**Comment by zaiweizhang** [comment]
Okay. Thanks a lot!

**Comment by imankgoyal** [comment]
Hi @syb7573330, I want to do a fair comparison with DGCNN, so can you please confirm what do you exactly mean by "the best results during the training process". It looks like a model is saved after every 5 epochs (https://github.com/WangYueFt/dgcnn/blob/master/tensorflow/part_seg/train_multi_gpu.py#L381). So is the best result on the test set out of all saved models is taken?
